### DISCUSSION TITLE
How can I trigger an SSE behavior upon data insertion?

### DISCUSSION THREAD

**Discussion by merrydance** [prisma/prisma#19612] [Q&A] [answered]
I have a scenario where the backend pushes data to the frontend through SSE, triggered by changes in a database table. Specifically, I have a message table, and when a production task is assigned, a notification is written to the message table and pushed to the relevant production personnel. When a new data record is inserted into the message table, it should trigger an SSE push. How can I listen for changes in the message table to trigger SSE behavior?

**Comment by nurul3101** [comment, marked as answer]
Hello @merrydance 👋 

This looks like a use case for [Pulse](https://www.prisma.io/data-platform/pulse), which is still in early access.
I would recommend you to have a look at this feature request
- https://github.com/prisma/prisma/issues/298

There are several useful suggestions in the feature request that should be helpful.

**Reply by merrydance** [reply]
thanks for your answer！
